tidy3d/plugins/autograd/functions.py

```diff
@@ -241,7 +241,10 @@ def pad(
         raise ValueError("Padding width must have one or two elements, got {pad_width.size}.")
 
     if any(any(p >= s for p in pad_width) for s in array.shape):
-        raise NotImplementedError("Padding larger than the input size is not supported.")
+        raise NotImplementedError(
+            "Padding larger than the input size is not supported. "
+            "The convolution kernel is probably too large for the input."
```

Comment:
can we display the relevant shapes here and maybe add more instruction on how to fix it?

Reply:
done